### DISCUSSION TITLE
How to download and install Krokiet on Mac?

### DISCUSSION THREAD

**Discussion by billytpy** [qarmin/czkawka#1399] [Q&A]
I just wanted to download and install Krokiet GUI on my Mac. I downloaded the file [mac_krokiet_gui](https://github.com/qarmin/czkawka/releases/download/8.0.0/mac_krokiet_gui), but it doesn't run. What should I do? Sorry for this stupid question, but I have no knowledge in programming.

**Comment by monsieurr** [comment]
1. Open the Terminal (**Cmd + Space -> Terminal -> Enter**)
2. Go to the place you have your **mac_krokiet_gui** (if it's in the "Downloads folder" then probably do `cd Downloads`) -> if you are not sure you can clic on the mac_krokiet_gui, Copy and paste it inside the terminal and you'll get the full path, then from this path you'll just have to remove mac_krokiet_gui and keep the rest (example : /Users/mac/Downloads/mac_krokiet_gui < remove mac_krokiet_gui)
3. Now that the Terminal is open write `chmod +x mac_krokiet_gui` and then click enter (note : you must be inside the folder you have your mac_krokiet_gui to do that)
4. Then in the same Terminal write `./mac_krokiet_gui`, this should open the application

**Reply by Ech-io** [reply]
Thanks for the guide! I'm now also in OP's position, and I can't get past the MacOS malware warning... any advice?

**Reply by llaa23** [reply]
Hello! How to fix "zsh: bad CPU type in executable: ./mac_krokiet_gui"?
I had installed Krokiet on my Mac before and this method worked until I accidentally deleted the Krokiet file in the download folder...

**Reply by monsieurr** [reply]
> Thanks for the guide! I'm now also in OP's position, and I can't get past the MacOS malware warning... any advice?

In general in the case of a system warning on MacOS, you can approve the software you want to open inside Sytem Settings > Privacy and Security then scroll down to see the application that you tried to open. You should be able to approve it there.

**Reply by monsieurr** [reply]
> Hello! How to fix "zsh: bad CPU type in executable: ./mac_krokiet_gui"? I had installed Krokiet on my Mac before and this method worked until I accidentally deleted the Krokiet file in the download folder...

Are you sure you have the right version? I'd guess mac_krokiet_gui is for Intel based Macs (so, older than 2020) and mac_krokiet_gui_arm should be for M chip based Macs (so, the newer ones)

**Reply by llaa23** [reply]
Thank you for your reply! I downloaded mac_krokiet_gui and my Mac is 2018.
I tried to approve the software inside Privacy and Security and after Step 4, which i write ./mac_krokiet_gui , in Terminal, 
it still appear "zsh: bad CPU type in executable: ./mac_krokiet_gui "
